configure

```diff
@@ -138,13 +138,23 @@ bindir=${bindir:-${prefix}/bin}
 libdir=${libdir:-${prefix}/lib}
 etcdir=${etcdir:-${prefix}/etc}
 mandir=${mandir:-${prefix}/share/man}
+compldir=
 
 if [ "$etcdir" = /usr/etc ]; then
     etcdir=/etc
 fi
 if [ -n "$sysconfdir" ]; then
     etcdir=$sysconfdir
 fi
+if [ ${prefix:0:10} = /usr/local ]; then
```

Comment:
We care only if it installs to the system directory (i.e. prefix is "/usr").  So it should be:

```
if [ "$prefix" = /usr ]; then
```